README.md

```diff
@@ -6,106 +6,20 @@ BuildBlast
-- `cd ~/buildblast-production/`
-- Start both servers (different terminal sessions)
-  - `./server`
-  - `cd www-content && ../www --config="../server_config.json"`
-
-Note: Confirmation emails will not work, since it's set to use the buildblast.com domain accounts.
+You may need to do some config setup due to some things being slightly broken, see [#189](https://github.com/crazy2be/buildblast/issues/189)
 
-<!-- TODO: This section is a bit... agressive. Might want to make it kinder sounding. (Kevin feels it's okay to be agressive though.) -->
-Development
------------
-Make a branch, develop your feature/bugfix, and send a pull request. A couple tips:
-- **Read your diffs**. We all change things casually when learning a system, and poke at various things, but you should revert these changes before committing (or stash them somewhere), to ensure your pull request is focused on one particular thing that needs changing, and to reduce the time it takes to review.
-- **Keep pull requests small and incremental**. If you want a really big, awesome feature, develop it one piece at a time, so that each of them can be evaluated individually, and so that we can all discuss how we want the feature to look *before* you've built the whole thing. This also allows other to build other awesome things you haven't though of yet from your pieces, and reduces the pain of merging to master (which is very quickly changing at this point).
-- **Format your code**.
-	- Things are a little looser on the JS side, but try to match the existing code. Always use tabs. Use semicolons. Braces on the same line. I'm sure you can figure it out :).
-	- On the go side, always run `go fmt buildblast/...` before submitting a pull request.
-
-Architecture (client side)
----------------
-The architecture is based on a few fairly simple structures, which communicate through a few objects (similar to http://yuml.me/d5f4fb7b). Specifically:
-- **container**: Just a div in the html, we use this to hook up the actual rendering. There are some hardcoded elements in this, which we hook up to to give THREE.js a place to render (amoung other things).
-- **Conn**: A wrapper for a WebSocket, used to communicate to the server.
-- **Scene**: A THREE.Scene, this holds all the meshes and models which THREE.js will render.
-- **ChunkManager**: Manages the chunks (of blocks), and handles adding and removing them from the THREE.Scene.
-- **EntityManager**: Manages the entities (players for now, in the future hopefully more) and add/removes them from the THREE.Scene.
-- **World**: Holds both the ChunkManager and EntityManager and provides useful logic to interface with them. Can directly expose blocks but also provides most of the helper functions you should need.
-- **PlayerUI**: Handles displaying the interface elements and rendering the scene. Shouldn't hold any game data, or be required for the game to run.
+To run the www server, enable authentication, and enable multiple worlds, see [WEBREADME.md]
```

Comment:
This link seems to be broken in the preview. But, that might just be from looking at the blob, and not viewing it as part of the GitHub pages

Reply:
The link was broken, you were correct. I just sort of assumed that would work without actually checking.

Reply:
I'm not sure why I expected that to work. [Clearly I shouldn't have](https://daringfireball.net/projects/markdown/syntax#link). I think I was misapplying mediawiki syntax to Mardown.

Reply:
Fixed in https://github.com/crazy2be/buildblast/commit/75ff7da95459d6315adc661d027f1b3fb1f8647f